src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+import duke.task.TaskList;
+import duke.task.Task;
+import duke.task.Deadline;
+import duke.task.Todo;
+import duke.task.Event;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+public class Storage {
+    private static String dukeFilePath;
+
+    /**
+     * Check whether duke.txt exists and read lines from that file.
+     * @param taskList TaskList that manages tasks.
+     */
+    public static void loadFromFile(TaskList taskList) {
+        try {
+            Path currentRelativePath = Paths.get("");
+            String s = currentRelativePath.toAbsolutePath().toString();
+
+            java.nio.file.Path path = java.nio.file.Paths.get(s, "src", "data");
```

Comment:
The fully qualified import could be removed here as well since it has already been imported above